.github/workflows/vast.yaml

```diff
@@ -63,14 +63,16 @@ jobs:
       - name: Install Dependencies
         run: |
           apt-get -qq update
-          apt-get -qqy install git libsnappy-dev libbrotli-dev zlib1g-dev wget liblz4-dev libzstd-dev ninja-build libpcap-dev libssl-dev libbenchmark-dev tcpdump lsb-release libflatbuffers-dev flatbuffers-compiler-dev libyaml-cpp-dev libsimdjson-dev gnupg2 gcc-8 g++-8 python3 python3-pip python3-venv jq apt-transport-https ca-certificates curl gnupg-agent software-properties-common
+          apt-get -qqy install git libsnappy-dev libbrotli-dev zlib1g-dev wget liblz4-dev libzstd-dev ninja-build libpcap-dev libssl-dev libbenchmark-dev tcpdump lsb-release libflatbuffers-dev flatbuffers-compiler-dev libyaml-cpp-dev libsimdjson-dev gnupg2 gcc-8 g++-8 python3 python3-pip python3-venv jq apt-transport-https ca-certificates curl gnupg-agent software-properties-common dirmngr
 
           # Need to specify backports, since spdlog also has a regular buster package.
           apt-get -qqy -t buster-backports install libspdlog-dev
 
           # clang 9 (c.f. https://apt.llvm.org/)
+          mkdir -p ~/.gnupg
+          echo "disable-ipv6" >> ~/.gnupg/dirmngr.conf
```

Comment:
Can you add a comment above this line _why_ we want to disable IPv6? This is not covered on `apt.llvm.org` as indicated by the comment.